### PR TITLE
Editorial mdlint and vale updates for the HowTo/Find-and-Connect directory

### PR DESCRIPTION
**Pull request by grantnoble**
The usual mdlint and vale updates (line lengths, where possible), eol spaces, passive to active voice, etc.)

### CODE REVIEW

File: docs/HowTo/Find-and-Connect/Managing-Peers.md [hunk 1]
```diff
@@ -1,49 +1,54 @@
...
-!!! example "Example log output"
-    `2018-10-16 12:37:35.479-04:00 | nioEventLoopGroup-3-1 | INFO  | NettyP2PNetwork | Successfully accepted connection from 0xa979fb575495b8d6db44f750317d0f4622bf4c2aa3365d6af7c284339968eef29b69ad0dce72a4d8db5ebb4968de0e3bec910127f134779fbcb0cb6d3331163c`
+    `2018-10-16 12:37:35.479-04:00 | nioEventLoopGroup-3-1 | INFO | NettyP2PNetwork | Successfully accepted connection from 0xa979fb575495b8d6db44f750317d0f4622bf4c2aa3365d6af7c284339968eef29b69ad0dce72a4d8db5ebb4968de0e3bec910127f134779fbcb0cb6d3331163c`
 
-## Limiting Remote Connections 
+## Limiting remote connections
 
-In private networks with a level of trust between peers, enabling the [remote connection limits](../../Reference/CLI/CLI-Syntax.md#remote-connections-limit-enabled)
-is unnecessary and disabling may increase the speed at which nodes can join the network.
+In private networks with a level of trust between peers, enabling the
```
Comment (NicolasMassart) [inline docs/HowTo/Find-and-Connect/Managing-Peers.md]: no fix to do here in this PR, but can we create a ticket to say that maybe we should add the info about why limiting remote connections and not only when not limiting?

File: docs/HowTo/Find-and-Connect/Specifying-NAT.md [hunk 2]
```diff
@@ -1,64 +1,72 @@
...
+Docker container, `AUTO` acts as if you specified [`NONE`](#none).
 
+## None
 
+Specify `NONE` to explicitly specify Besu does not use NAT. If the NAT method is `NONE`, Besu
+ignores any values specified in the [`--p2p-host`](../../Reference/CLI/CLI-Syntax.md#p2p-host),
+[`--p2p-port`](../../Reference/CLI/CLI-Syntax.md#p2p-port),
+[`--rpc-http-host`](../../Reference/CLI/CLI-Syntax.md#rpc-http-host), and
+[`--rpc-http-port`](../../Reference/CLI/CLI-Syntax.md#rpc-http-port) options, with the default
+values advertised instead.
```
Comment (NicolasMassart) [inline docs/HowTo/Find-and-Connect/Specifying-NAT.md]: "and the default
values are advertised instead." ?

File: docs/HowTo/Find-and-Connect/Bootnodes.md [hunk 3]
```diff
@@ -1,40 +1,48 @@
+---
 description: Configuring bootnodoes
-<!--- END of page meta data -->
+---
 
 # Bootnodes
 
-Bootnodes are used to initially discover peers. A bootnode is a regular node a node uses to discover nodes.
+Using Bootnodes is method for initially discovering peers. A bootnode is a regular node a node uses
```
[the remark this answers is not in the document]
Comment (grantnoble) [reply docs/HowTo/Find-and-Connect/Bootnodes.md]: You're right, I definitely missed "a". Reworded as follows:
"Using Bootnodes is a method for initially discovering peers. Bootnodes are regular nodes used to discover other nodes."

File: docs/HowTo/Find-and-Connect/Bootnodes.md [hunk 4]
```diff
@@ -1,40 +1,48 @@
...
+    nodes are always connected (using static nodes).
 
-For mainnet, Rinkeby, Ropsten, and Görli, Hyperledger Besu predefines a list of enode URLs.  
+    To find peers, configure one or more bootnodes as described below. To configure a specific set
+    of peer connections, use [static nodes](Static-Nodes.md).
+
+## MainNet and public testnets
+
+For MainNet and the Rinkeby, Ropsten, and Görli testnets, Hyperledger Besu predefines a list of
+enode URLs.
```
[the remark this answers is not in the document]
Comment (grantnoble) [reply docs/HowTo/Find-and-Connect/Bootnodes.md]: Done.
"... Hyperledger Besu predefines a list of enode URLs and uses this list automatically when you specify the
[`--network`](../../Reference/CLI/CLI-Syntax.md#network) option.

File: docs/HowTo/Find-and-Connect/Bootnodes.md [hunk 5]
```diff
@@ -1,40 +1,48 @@
...
 !!! example
+
     ```bash
     besu --genesis-file=privateNetworkGenesis.json --data-path=nodeDataPath --bootnodes=enode://c35c3ec90a8a51fd5703594c6303382f3ae6b2ecb99bab2c04b3794f2bc3fc2631dabb0c08af795787a6c004d8f532230ae6e9925cbbefb0b28b79295d615f@127.0.0.1:30303
-    ``` 
+    ```
 
-The default host and port for P2P peer discovery is `127.0.0.1:30303`. Use the [`--p2p-host`](../../Reference/CLI/CLI-Syntax.md#p2p-host) 
-and [`--p2p-port`](../../Reference/CLI/CLI-Syntax.md#p2p-port) option to specify a host and port.
+The default host and port for P2P peer discovery is `127.0.0.1:30303`. To specify a different host
```
[the remark this answers is not in the document]
Comment (grantnoble) [reply docs/HowTo/Find-and-Connect/Bootnodes.md]: Done.

File: docs/HowTo/Find-and-Connect/Managing-Peers.md (hunk 1 above)
Comment (grantnoble) [reply docs/HowTo/Find-and-Connect/Managing-Peers.md]: Done.

File: docs/HowTo/Find-and-Connect/Specifying-NAT.md (hunk 2 above)
Comment (grantnoble) [reply docs/HowTo/Find-and-Connect/Specifying-NAT.md]: Broke it up into two sentences. 
"... options. Besu advertises the default values instead."

**Pull request merged**